From 5c67195bdf0f8cd50c835ebe9e75baaa175904f6 Mon Sep 17 00:00:00 2001
From: John Levon <john.levon@joyent.com>
Date: Fri, 30 Aug 2019 14:50:42 +0000
Subject: [PATCH] OS-7973 fbt entry probes incorrectly placed due to
 -fshrink-wrap Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com> Reviewed
 by: Robert Mustacchi <rm@joyent.com> Approved by: Robert Mustacchi
 <rm@joyent.com>

---
 tun/Makefile | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/tun/Makefile b/tun/Makefile
index a28d5b2..4941c32 100644
--- a/tun/Makefile
+++ b/tun/Makefile
@@ -18,13 +18,15 @@
 #
 # CDDL HEADER END
 #
-# Copyright (c) 2012, Joyent, Inc.  All rights reserved.
+# Copyright 2019 Joyent, Inc.
 #
 
-VER =	tun-1.3
+VER = tun-1.3
 
 include ../Makefile.defs
 
+CFLAGS.64 += -fno-shrink-wrap
+
 BUILD32 =
 BUILD64 = yes
 
-- 
2.21.0

